.github/workflows/ci.yml

```diff
@@ -41,123 +36,95 @@ jobs:
         if: contains(matrix.os, 'ubuntu')
 
       - name: install stable toolchain
-        uses: actions-rs/toolchain@v1
+        uses: dtolnay/rust-toolchain@v1
         with:
-          toolchain: 1.68
+          toolchain: stable
```

Comment:
Sorry for the confusion but we don't use `stable`, so please revert 008e115. The move to explicit versions was very much on purpose and was discussed at length. The original PR for that was #76.

Cleaning up the CI, along with better comments to explain things, has been on my todo for months, but unfortunately I've been too busy to get to it. So sorry for that.

You can leave it at `1.68` or bump it up all the way to `1.70` if you wish to fix the resulting clippy errors.